input_files/LoadProfiles/test_read.py

```diff
@@ -0,0 +1,12 @@
+filename = 'Load8760_norm_Albuquerque_FastFoodRest.dat'
```

Comment:
this looks like a file that can be deleted?

Reply:
Agreed; this has been updated in commit 7105de7f327ea3518cd9dd6ce00c541cd09b6c31